src/olympia/addons/migrations/0029_delete_appsupport.py

```diff
@@ -0,0 +1,16 @@
+# Generated by Django 2.2.17 on 2021-01-14 18:05
```

Comment:
Actually, we'll have to land this in a following tag.  I'll put it out before merging.

Reply:
Could you move that file to another PR then?

Reply:
I will.